c2ea.py

```diff
@@ -2,12 +2,11 @@
 import tkinter as tk
 from tkinter import filedialog
 
-
 def show_exception_and_exit(exc_type, exc_value, tb):
-    import traceback
-    traceback.print_exception(exc_type, exc_value, tb)
-    input("Press Enter key to exit.")
-    sys.exit(-1)
+  import traceback
```

Comment:
I think this block should stay at 4 indent (And throughout-- I think we should keep 4 space indents).

Reply:
I'll handle this